ts/Extensions/Annotations/Types/FibonacciTimeZones.ts

```diff
@@ -83,11 +86,19 @@ class FibonacciTimeZones extends CrookedLine {
     ): Function {
         return function (target: any): PositionObject {
             const annotation = target.annotation;
+
             let points = annotation.points;
             // Offset between the first and the second line
-            const deltaX = points.length > 1 ? points[1].x - points[0].x : 0,
+
+            const xAxis = points[0].series.xAxis,
```

Comment:
Because of that nasty casting `target: any` we are losing all TS here.
Are you sure that `points` always exist?

Reply:
This is a huge TS refactoring. This should be done not only for the FibonacciTimeZones annotation, but for all others too. In my opinion, TS refactor here should be in a separate ticket. What do you think?

Reply:
I'm ok with that ;) 
It seems like all other annotations have the same TS issue and they are working fine. So let's leave it for now.